horovod/common/parameter_manager.cc

```diff
@@ -176,11 +152,14 @@ void ParameterManager::Update(const std::vector<std::string>& tensor_names, int6
   if (sample_ >= SAMPLES) {
     std::sort(scores_, scores_ + SAMPLES);
     double med_score = scores_[SAMPLES / 2];
-    Tune(med_score);
+    if (Tune(med_score))
```

Comment:
Looks like we can simplify to `return Tune(...)`

Reply:
Yes.